### ISSUE TITLE
Lambda not getting invoked from S3 bucket notification

### ISSUE THREAD

**Issue by ambreenshameem** [localstack/localstack#2776]
I am a beginner and have started to explore localstack for testing AWS services. 
I am trying to put a file in S3 bucket, lambda is triggered on add file event in s3 bucket, and lambda process the file and save it’s content in dynamo db.
I am able to create bucket, create function, create table and upload file but after uploading the file in s3 bucket, the lambda is not getting triggered and my table remains empty. 
i am attaching the docker file . When i look into the docker dashboard i see the localstack server up and running but it doesn't display the steps i am performing in localstack for ex: creating bucket and table.
Is there a way i can validate my lambda running in the docker and is talking to other services?

docker-compose.yml file --
```
 version: '2.1'

services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-awslocalstack}"
    image: localstack/localstack
    ports:
      - "4566-4599:4566-4599"
      - "${PORT_WEB_UI-8081}:${PORT_WEB_UI-8080}"
    environment:
      - SERVICES=${SERVICES- }
      - DEBUG=${DEBUG- }
      - DATA_DIR=${DATA_DIR- }
      - PORT_WEB_UI=${PORT_WEB_UI- }
      - LAMBDA_EXECUTOR=docker              #${LAMBDA_EXECUTOR- }
      - LAMBDA_REMOTE_DOCKER=true
      - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
      - DOCKER_HOST=unix:///var/run/docker.sock
      - HOST_TMP_FOLDER=${TMPDIR}
      - LOCALSTACK_HOSTNAME=${LOCALSTACK_HOSTNAME- }
    volumes:
      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"
```

commands i am running to achieve the goal:

bucket creation:
```
aws s3 mb s3://person-bucket --endpoint-url http://localhost:4572
```

table creation:
```
aws dynamodb create-table --table-name PersonDetails --attribute-definitions AttributeName=Id,AttributeType=N --key-schema AttributeName=Id,KeyType=HASH --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --endpoint-url=http://localhost:4569
```

Lambda function:
```
aws lambda create-function --endpoint-url=http://localhost:4574 --function-name samplelambda --runtime java8 --handler lambda.S3EventHandler --zip-file fileb://C:\Workspace\AWS_LocalStack\localstack-demo\target\lambdademo-1.0.0.jar --role arn:aws:iam::1234567:role/anything --environment Variables="{PROFILE=LOCAL,AWS_REGION=us-east-1}"
```

notification trigger :
```
aws s3api put-bucket-notification-configuration --bucket person-bucket --notification-configuration file://notification.json --endpoint-url http://localhost:4572
```

dynamodb scan:
```
aws dynamodb scan --table-name PersonDetails --endpoint-url http://localhost:4569
```

copy samplejson in s3:
```
aws s3 cp sample.json s3://person-bucket --endpoint-url=http://localhost:4572
```

**Comment by whummer** [comment]
Thanks for reporting @ambreenshameem . Can you please share the content of your `notification.json` file?

**Comment by ambreenshameem** [comment]
@whummer   -- Notification.json

{
  "LambdaFunctionConfigurations": [
    {
      "Id": "123456788",
      "LambdaFunctionArn": "arn:aws:lambda:us-east-1:000000000000:function:samplelambda",
      "Events":[
        "s3:ObjectCreated:*"
      ]
    }
  ]
}

**Comment by ambreenshameem** [comment]
@whummer - is somebody looking into this issue and help me resolve it. I am totally stuck and unable to proceed.

**Comment by ambreenshameem** [comment]
any response is greatly appreciated!!

**Comment by whummer** [comment]
Thanks for the update @ambreenshameem . This should be fixed in #2795 - can you please do a `docker pull localstack/localstack` and give it another try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks

**Comment by ambreenshameem** [comment]
OMG!! big thank you. It works perfectly now.
